tools/EventClients/lib/python/xbmcclient.py

```diff
@@ -279,7 +282,7 @@ def __init__(self, devicename=None, icon_type=ICON_NONE, icon_file=None):
         self.append_payload( format_uint32 (0) ) # reserved1
         self.append_payload( format_uint32 (0) ) # reserved2
         if icon_type != ICON_NONE and icon_file:
-            self.append_payload( file(icon_file).read() )
+            self.append_payload( open(icon_file, 'rb').read() )
```

Comment:
Probably better to use python's `with` statement here to ensure the file is always closed